tests/modules/nf-core/quilt/quilt/main.nf

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { QUILT_QUILT                   } from '../../../../../modules/nf-core/quilt/quilt/main.nf'
+include { QUILT_QUILT as QUILT_OPTIONAL } from '../../../../../modules/nf-core/quilt/quilt/main.nf'
+
+
+    // input sequencing data (bam)
+    bam = [
+        file('https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/quilt/NA12878.haplotagged.1.0.bam', checkIfExists: true),
```

Comment:
Are these files going to be moved out of delete_me?

Reply:
As of now, the files will continue to reside in the "delete_me" directory until a consensus is reached on the most suitable method for organizing files used to test a module. It's worth noting that upon the publication of the complete #phaseimpute pipeline, we anticipate having our dedicated repository to manage test data. At that point, we will adjust the file storage accordingly.

Reply:
Ok, just wanted to highlight that delete_me isn't for long-term storage.